contracts/libs/Owned.sol

```diff
@@ -13,5 +13,9 @@ contract Owned {
 
   function isOwner(address addr) public returns(bool) { return addr == owner; }
```

Comment:
The reason I've keept it is that it is convenient to use from outside the EVM. I.E. if you want a simple way of checking the owner from javascript.